src/SingleProject/Resizetizer/test/UnitTests/GenerateSplashAndroidResourcesTests.cs

```diff
@@ -83,7 +82,32 @@ public void SplashScreenResectsAlias(string alias, string outputImage)
 			var success = task.Execute();
 			Assert.True(success, LogErrorEvents.FirstOrDefault()?.Message);
 
-			AssertImageFile($"@drawable/{outputImage}");
+			AssertImageFile("maui_splash_image.xml", _drawable, $"@drawable/{outputImage}");
+			AssertImageFile("maui_splash_image_v31.xml", _drawable_v31, $"@drawable/{outputImage}");
+		}
+
+		void AssertColorsFile(string expectedFilename, string color)
+		{
+			var expectedXml = File.ReadAllText($"testdata/androidsplash/" + expectedFilename)
+				.Replace("{maui_splash_color}", color, StringComparison.OrdinalIgnoreCase);
+
+			var actual = XElement.Load(_colors);
+			var expected = XElement.Parse(expectedXml);
+
+			Assert.True(XNode.DeepEquals(actual, expected), $"{_colors} did not match:\n{actual}");
+		}
+
+		void AssertImageFile(string expectedFilename, string actualFilename, string image, string width = "108", string height = "108")
```

Comment:
If the images here are resized to 108dp x 108dp, do we also resize the contents inside the app?

If I put a 2048x2048 image, will I get full size on iOS, but a scaled image on Android? To avoid the `.apk` size growing?

Reply:
You won't as it stands because previous versions of android use the full size. You can still control the size using base size, but it uses the original size

If you check previous android versions it fills the whole screen.

Reply:
We technically could make everything consistent and always have a 108dp splash image size...

Reply:
More issues here: https://github.com/dotnet/maui/issues/10604#issuecomment-1274519803